www/front_src/src/Resources/Graph/Performance/Lines/RegularLine.tsx

```diff
@@ -30,24 +35,76 @@ const RegularLine = ({
   metric,
   lineColor,
   unit,
+  lines,
   yScale,
   xScale,
   areaColor,
   transparency,
   graphHeight,
+  resourceType,
 }: Props): JSX.Element => {
+  const openModalAD = useAtomValue(openModalADAtom);
+
+  const strokeWidth =
+    equals(metric, 'connection_lower_thresholds') ||
+    equals(metric, 'connection_upper_thresholds')
+      ? 0.1
+      : 0.8;
+
+  const isLegendClicked = lines?.length <= 1;
+  const isHighlight = highlight || isLegendClicked ? 2 : strokeWidth;
+
+  interface PropsIsOnline {
+    maxDistance: number;
+    pointX: number;
+    pointX1: number;
+    pointX2: number;
+    pointY: number;
+    pointY1: number;
+    pointY2: number;
+  }
+
+  const isOnLine = ({
+    pointX,
+    pointY,
+    pointX1,
+    pointY1,
+    pointX2,
+    pointY2,
+    maxDistance,
+  }: PropsIsOnline): boolean => {
+    const dxL = pointX2 - pointX1;
+    const dyL = pointY2 - pointY1;
+    const dxP = pointX - pointX1;
+    const dyP = pointY - pointY1;
+
+    const squareLen = dxL * dxL + dyL * dyL;
+    const dotProd = dxP * dxL + dyP * dyL;
+    const crossProd = dyP * dxL - dxP * dyL;
```

Comment:
Can you use words please to better understand your code?

Reply:
it is a formula of Darius Bacon which determines if a point c between two points a and b ,

Reply:
it is a formula of Darius Bacon which determines if a point c between two points a and b : code en python https://stackoverflow.com/questions/328107/how-can-you-determine-a-point-is-between-two-other-points-on-a-line-segment/328122#328122   for js : https://stackoverflow.com/a/34474547 👀